GDScript/hello_in_childNode.gd

```diff
@@ -0,0 +1,7 @@
+extends Node
+
+onready var childNode_label = null
```

Comment:
code can be simplified by changing the 'null' in line 3 to '$Label' and deleting the line 6